test/tests/selftest/pbs_test_revert_site_hooks.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2019 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.selftest import *
+
+
+class TestRevertSiteHook(TestSelf):
+    """
+    Contains test to delete hooks in Setup() except cgroups hook
+    """
+
+    def test_verify_delete_hooks_in_setup(self):
+        """
+        Test if cgroups hook exists at the end of the test case
+        """
+        self.server.revert_to_defaults(self)
+        hooks = [h['id'] for h in self.server.status(HOOK)]
+        for h in hooks:
```

Comment:
If in future we have more than one default hook enabled, this test will fail.
we do not need a for loop here - 
just check for the existence of 'pbs_cgroup' in hooks with if 'pbs_cgroups' in hooks